docs/telemetryData.md

```diff
@@ -18,98 +19,100 @@ As new features and pages roll out, percentages  will adjust.
 
 |Project|Percentage|
 |:---|:---:|
-|Navigation View|66.2%|
-|Blank|16.7%|
-|Horizontal Navigation View|9.3%|
-|MenuBar|7.8%|
+|Navigation View|69.3%|
+|Blank|13.8%|
+|Horizontal Navigation View|10.1%|
+|MenuBar|6.8%|
 
 ### Framework
 
 |Framework Type|Percentage|
 |:---|:---:|
-|CodeBehind|52.6%|
-|MVVM Basic|17.3%|
-|Prism|11%|
-|Caliburn.Micro|9.8%|
-|MVVM Light|9%|
-|MVVMToolkit|0.2%|
+|MVVMToolkit|48.8%|
+|CodeBehind|31.6%|
+|Prism|8.2%|
+|MVVM Basic|5.6%|
+|Caliburn.Micro|3%|
+|MVVM Light|2.8%|
 
 ### Pages
 
 |Pages|Percentage|
 |:---|:---:|
-|Blank|31%|
-|Settings|12.9%|
-|ListDetail|7.4%|
-|Content Grid|6.7%|
-|Web View|6.3%|
-|DataGrid|5.7%|
-|MediaPlayer|3.9%|
-|ImageGallery|3.7%|
-|TreeView|2.9%|
-|TabView|2.9%|
-|Tabbed / Pivot|2.7%|
-|Chart|2.7%|
-|Two Pane View|2.2%|
-|Telerik Data Grid|2.1%|
-|Map|2%|
-|Camera|1.7%|
+|Blank|32.2%|
+|Settings|13.7%|
+|Master/Detail|8.2%|

```

Comment:
should it be Master/Detail or ListDetail?

Reply:
It should be ListDetail, we renamed the page in the last version, but there are a bunch of templates where the telemetryname was set to MasterDetail. This will be fixed once 4.0 is released.